deploy.py

```diff
@@ -174,14 +171,18 @@ def deploy(release, name=None):
         "mybinder",
         "--cleanup-on-fail",
         "--create-namespace",
-        "-f",
-        os.path.join("config", release + ".yaml"),
-        "-f",
-        os.path.join("secrets", "config", "common.yaml"),
-        "-f",
-        os.path.join("secrets", "config", release + ".yaml"),
     ]
 
+    # common config files
+    config_files = glob.glob(os.path.join("config", "common", "*.yaml"))
```

Comment:
```suggestion
    config_files = sorted(glob.glob(os.path.join("config", "common", "*.yaml")))
```
IIRC glob doesn't guarantee ordering. Doesn't matter at the moment, but might do if in future other configs are added to common.